sdcm/nemesis.py

```diff
@@ -3341,8 +3341,10 @@ def is_virtual_tables_get_snapshot():
                 return ComparableScyllaVersion(self.target_node.scylla_version) >= "6.3.0-dev"
 
         filter_func = self.cluster.is_table_has_no_sstables if is_virtual_tables_get_snapshot() else None
+        # Stop taking snapshots of MVs when snapshotting a base table - presented in
+        # https://github.com/scylladb/scylladb/pull/21433/commits/9645a0414dbf4f41e0ce612b485b09c16a408f42
         ks_cf = self.cluster.get_any_ks_cf_list(
-            db_node=self.target_node, filter_empty_tables=False, filter_func=filter_func)
+            db_node=self.target_node, filter_empty_tables=False, filter_func=filter_func, filter_out_mv=True)
```

Comment:
Need to make it be conditional based on the scylla version using `ComparableScyllaVersion` like above.

SCT master branch is used for multiple scylla versions, not only latest ones.